assembly/assembly-main/src/assembly/bin/docker.sh

```diff
@@ -121,6 +129,9 @@ init() {
     export CHE_WORKSPACE_STORAGE_CREATE_FOLDERS=false
   fi
 
+  # Ensure that the user has permissions for CHE_HOME and CHE_DATA
+  sudo chown -R ${CHE_USER}:docker ${CHE_HOME}
```

Comment:
If we don't call chown, che will start only if we do that manually before starting the che server.